File: YesTask/app/src/main/java/com/felix/yestask/code/model/RMCategory.java
package com.felix.yestask.code.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by user on 21/09/2017.
 */

public class RMCategory extends RealmObject{

    @PrimaryKey
    private int id;
    private String name;
    private Integer color;

    public RMCategory(){

    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getColor() {
        return color;
    }

    public void setColor(Integer color) {
        this.color = color;
    }
}

File: YesTask/app/src/main/java/com/felix/yestask/code/model/RMCategory.kt
package com.felix.yestask.code.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

/**
 * Created by user on 03/10/2017.
 */

class RMCategory : RealmObject(){

    @PrimaryKey
    var id: Int = 1
    var name : String? = null
    var color : Integer? = null
}